core/Dockerfile

```diff
@@ -262,6 +262,9 @@ EOF
         touch /var/www/MISP/.git/ORIG_HEAD && chmod 0600 /var/www/MISP/.git/ORIG_HEAD && chown www-data:www-data /var/www/MISP/.git/ORIG_HEAD
 EOF
 
+    # Include rsyslog to support syslogger
+    RUN apt-get update ; apt-get install -y rsyslog
```

Comment:
This needs to be moved where the layer is built

Reply:
Do you mean as part of the `apt-get install` at https://github.com/MISP/misp-docker/blob/master/core/Dockerfile#L169   ?

Reply:
Correct